CantoboardFramework/Keyboard/FeedbackProvider.swift

```diff
@@ -21,7 +21,7 @@ class FeedbackProvider {
             AudioServicesPlaySystemSound(Self.clickPress)
         case .backspace, .deleteWord:
             AudioServicesPlaySystemSound(Self.deletePress)
-        case .keyboardType(_), .space, .newLine:
+        case .keyboardType(_), .space, .newLine, .shift(_), .nextKeyboard:
```

Comment:
LGTM

Reply:
Note that the `TouchHandler.swift` file was also edited in order to achieve this.